pxtblocks/fields/field_tilemap.ts

```diff
@@ -262,6 +262,11 @@ namespace pxtblockly {
         private parseBitmap(newText: string) {
             if (!this.blocksInfo) return;
 
+            if (newText) {
+                // backticks are escaped inside markdown content
+                newText = newText.replace(/&#96;/g, "`");
```

Comment:
can we do this in tutorial.js when the jres is parsed?

Reply:
actually, i'm not sure where the escape is happening. I tried moving this to tutorial.ts and marked.tsx. I think it might be the decompiler somehow? anyways, I think this is the best place for it.